src/main/java/com/ongvida/api/controller/AlunoController.java

```diff
@@ -0,0 +1,91 @@
+package com.ongvida.api.controller;
+import java.util.List;
+import java.util.Optional;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import com.ongvida.domain.model.Aluno;
+import com.ongvida.domain.repository.AlunoRepository;
+import com.ongvida.domain.service.CadastroAlunoService;
+@RestController
+@RequestMapping("/alunos")
+
+public class AlunoController {
+	
+	@Autowired
+	private AlunoRepository alunoRepository;
+	@Autowired
+	private CadastroAlunoService cadastroAlunoService;
+	
+	@GetMapping
+	public List<Aluno> listar(){
+		
+		return alunoRepository.findAll();
+		}
+	@GetMapping("/listar")
+	public List<Aluno> listarPorNome(){				
+		return alunoRepository.findByNome("alunoAtualizado");
+	}
+	
+	@GetMapping("/listarcontaining")
+	public List<Aluno> listarPorNomeContaining(){
+		return alunoRepository.findByNomeContaining("tu");
+	}
+	
+	@GetMapping("/{alunoId}")
+	public ResponseEntity<Aluno> buscar(@PathVariable Long alunoId) {
+		Optional<Aluno> aluno = alunoRepository.findById(alunoId);
```

Comment:
```suggestion
	@GetMapping("/{studentId}")
	public ResponseEntity<Student> findById(@PathVariable Long studentId) {
		Optional<Student> student = studentRepository.findById(studentId);
```

Como eu disse, não é pra ter código em português. Quando quiser usar uma palavra nova ou sinônima de outra recomendo traduzir no tradutor e jogar no  

https://www.thesaurus.com/

Assim, você vai ampliar o vocabulário com os possíveis signigificados da palavra e ver outros sinônimos pra usar como nome de classes e objetos :+1: